docusaurus/docs/extensions/advanced/safe-mode.md

```diff
@@ -0,0 +1,5 @@
+# Safe Mode
+
+If you have developed and installed an Extension that has an issue that blocks the load of Rancher Dashboard, you might get "blocked" of accessing and navigating around Rancher Dashboard. In that case, there's a route path called `safeMode` that prevent's the load of installed Extensions, therefore regaining access to Rancher Dashboard.
+
+To enable this safe mode, you just need to enter your Rancher Dashboard in the following way: `https://<- YOUR RANCHER BASE URL ->/safeMode`.
```

Comment:
We're you able to verify that this works? It seems to yield 404s for me.

<img width="746" alt="image" src="https://github.com/rancher/dashboard/assets/55104481/74c23cf9-5e87-499e-af37-1ec60cc0c6c6">

Reply:
I tried it with different casing and the suffix on the end of an existing page and still get a page not found

<img width="625" alt="image" src="https://github.com/rancher/dashboard/assets/55104481/4ce81ad2-2301-44d0-b430-6c1aad62dbd4">

Reply:
We figured out the issues and made a small change to improve things elsewhere
https://github.com/rancher/dashboard/pull/9136